client.go

```diff
@@ -85,7 +103,10 @@ func (c *Client) Call(params, resp interface{}) error {
 	if rt.method == "" {
 		return errgo.Newf("type %T has no httprequest.Route field", params)
 	}
-	reqURL, err := appendURL(c.BaseURL, rt.path)
+	if rt.path == "/" {
```

Comment:
Hmm, this makes me uncomfortable. Perhaps we should clarify the semantics of appendURL to make appendURL(x, "/") a no-op for any x. Or perhaps explicitly allow a path of "." to mean that. Or allow an empty path in parseRouteTag and provide checks in its callers as appropriate.